### DISCUSSION TITLE
Feature: Something like wp-playground but for Joomla

### DISCUSSION THREAD

**Discussion by alexandreelise** [joomla/joomla-cms#43744] [2. Cold Features]
Hello Super Joomlers!

Would it be great to have something like https://github.com/WordPress/wordpress-playground but for Joomla which would lower the barrier for new users to try Joomla without having a server/cloud/hosting. Ability to try Joomla directly in the browser using Web Assembly and SQLite. No external dependency. Works offline too AFAIK. Waiting for your thoughts and feedback Super Joomlers!

**Comment by alikon** [comment]
Drupal too https://mglaman.dev/blog/running-drupal-edge-webassembly

**Reply by alexandreelise** [reply]
Thx Nicola you are always on the cutting edge technologies @alikon
